### PR TITLE
fix: textarea and textfield improvements

### PR DESCRIPTION
**Pull request by victorscania**
**Describe pull-request**  
Updated textarea and textfield with new functionality

**Solving issue**  
Fixes: [AB#1381](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1381) and [AB#1781](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1781)

**How to test**  
Go to the generated storybook link
1781: Go to textarea, use the controler for rows and test if the size changes as you change rows.
1381: Go to textfield, change the controler for min width. resize the demo-contianer with inline with and se if the field cna shrink.

**Link for test**
https://task-1381.d1g8v8mrkx992n.amplifyapp.com/

### CODE REVIEW

**Comment by areblk** [comment]
**Text field nominwidth = true works as expected.**

**Suggested fixes for textarea**
I expected text area to have a "default" height that would apply when there is no row-attribute or when `rows`-argument is left blank, but it didn't. Suggestions for improvement:
1. When I delete the row-attribute in inspector, text area is slightly shorter than it used to be. 
2. I'd like to be able to leave `rows`-argument empty without getting an error message. (I've seen other components using the ternary operator to set value to default when there is no other value)

**Review by gaonita — APPROVED** [review]
confirmed it works as intended :)

**Comment by theJohnnyMe** [comment]
I think your PR went to rebase hell, right? I will try to clean it up :)

**Comment by theJohnnyMe** [comment]
@areblk and @victorscania  - I fixed the issue caused by rebase. Please double-check if everything looks ok. :)

File: components/src/components/textfield/textfield.tsx [hunk 1]
```diff
@@ -86,9 +89,11 @@ export class Textfield {
   }
 
   render() {
+    console.log(this.nominwidth);
```
Comment (theJohnnyMe) [inline components/src/components/textfield/textfield.tsx:92]: I just noticed this one, it cab be dropped, right?

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]
Tiny thing from my side, others look good.

**Review by theJohnnyMe — APPROVED** [review]
Great work! Looks good to me! Squash and merge it!  👍

**Review by areblk — APPROVED** [review]

**Pull request merged**